7_react/react1/week3/todo-app/src/components/TodoForm/utilities.js

```diff
@@ -0,0 +1,6 @@
+let nextId = 0;
+export function generateId() {
```

Comment:
This approach is not safe, it would work only till the user won't refresh the page. 

Lets say that you would store the user's data, 
1. The user comes to the page and will create a todo "some todo" id: 0 (nextId current value)
2. He decided that he will the rest later so he closes the browser, then he comes back after some time and create another todo, but because the previous instance of the page was terminated the `nextId` start from 0 again

There are other ways how to achieve a uniq ID: 
1. using timestamp, you can use Date API to get the current timestamp, to make it more robust, you can combine it with the todo description
2. Math.random method
3. maybe overkill for the app of this scale, but the safest approach is using uuid library https://github.com/uuidjs/uuid